Src/Plugins/Tests.Security/Tests.Security.csproj

```diff
@@ -13,6 +13,11 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
I don't think we need this group.

Reply:
fixed.

---
In reply to: [578079080](https://github.com/microsoft/sarif-pattern-matcher/pull/228#discussion_r578079080) [](ancestors = 578079080)